__tests__/__utils__/services/database.ts

```diff
@@ -39,4 +39,5 @@ interface AbstractUuid<Typename extends string> {
   id?: number
   alias?: string
   oldAlias?: string
+  instance?: 'de' | 'en' | 'ta'
```

Comment:
We already have an Instance type for that: https://github.com/serlo/serlo.org-cloudflare-worker/blob/03cf8e731eebb97c41b1458b75c04921868d820c/src/utils/index.tsx#L34